src/index.js

```diff
@@ -5,6 +5,11 @@ const fs = require("node:fs/promises");
 const { program } = require("commander");
 const { version, name, description } = require("../package.json");
 const OpenAI = require("openai");
+const toml = require("smol-toml");
+const path = require("path");
+const os = require("os");
+
+const configFilePath = path.join(__dirname, "../.codeshift.toml");
```

Comment:
Should look in the user's home directory `~`

See https://stackoverflow.com/questions/9080085/node-js-find-home-directory-in-platform-agnostic-way

https://nodejs.org/docs/latest-v20.x/api/os.html#oshomedir